openslides_backend/action/actions/motion/create_forwarded.py

```diff
@@ -0,0 +1,113 @@
+import time
+from typing import Any, Dict
+
+from ....models.models import Motion
+from ....shared.exceptions import ActionException
+from ....shared.patterns import Collection, FullQualifiedId
+from ...mixins.create_action_with_dependencies import CreateActionWithDependencies
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..agenda_item.agenda_creation import CreateActionWithAgendaItemMixin
+from ..agenda_item.create import AgendaItemCreate
+from ..list_of_speakers.create import ListOfSpeakersCreate
+from ..list_of_speakers.list_of_speakers_creation import (
+    CreateActionWithListOfSpeakersMixin,
+)
+from ..motion_submitter.create import MotionSubmitterCreateAction
+from .sequential_numbers_mixin import SequentialNumbersMixin
+from .set_number_mixin import SetNumberMixin
+
+
+@register_action("motion.create_forwarded")
+class MotionCreate(
```

Comment:
Please rename

Reply:
Done. And renamed this and the test class into MotionCreateForwarded.